pybtp/btp.py

```diff
@@ -507,7 +507,7 @@ def gap_wait_for_disconnection(timeout=30):
     stack.gap.wait_for_disconnection(timeout)
 
 
-def gap_adv_ind_on(ad={}, sd={}):
+def gap_adv_ind_on(ad={}, sd={}, duration=4294967295, own_addr_type=OwnAddrType.le_identity_address):
```

Comment:
I would create a constant value for max duration and use it here.

Reply:
Agree. Where to add it? (I'm still new to the codebase). defs.py seems like the correct place for this, but it is autogenerated from zepyhr sources, types.py has mostly enum style items.

Reply:
I think types.py is fine.

Reply:
Ok. I've added a AdDuration class with a forever constant.